packages/next/shared/lib/router/router.ts

```diff
@@ -1092,7 +1092,7 @@ export default class Router implements BaseRouter {
     // The build manifest needs to be loaded before auto-static dynamic pages
     // get their query parameters to allow ensuring they can be parsed properly
     // when rewritten to
-    let pages: any, rewrites: any
+    let pages: string[], rewrites: any
```

Comment:
> `any`

TypeScript would have caught this crash were it not for the `any` and `!`.

cc @delbaoliveira since we'd been chatting about this at Prisma Day - happy to help clean up these types! 🙂